ci/cd/run.sh

```diff
@@ -31,3 +31,4 @@ make ci-cd-upgrade-cluster
 make test-remote || true
 make test-remote-looker
 make test-remote-ui
+make test-browser
```

Comment:
ignorant question before understanding most of what's going on here : )

How would you describe the responsibility of `make test-remote-ui` vs `make test-browser`? Or is this really part of `[wip]` and should keep this question to myself until this is ready for review? :D

Reply:
@jgehrcke `make test-remote-ui` doesn't (well didn't when I started this work) do browser based testing. It uses playwright to get creds so that it can make api calls. `make test-browser` is actual browser testing, it has zero dependencies on the rest of the code base and is completely stand alone, it's also is testing against multiple browsers as it's using the Playwright Test runner which is designed for that - where as the mocha setup that test-remote-ui uses can't do multi-browser testing without getting hacky.

note: There have been some browser tests put into test-remote-ui in the last week which I'll be helping to move accross to test-browser - should be very straight forward and they get multi-browser testing for free. Also, in a previous version of this PR/change I had renamed `test-remote-ui` to `test-remote-ui-api` to make it more clear what it's for but dropped that for the time being as it got a bit out of scope.